src/context.rs

```diff
@@ -157,22 +157,20 @@ pub fn dc_context_new(
         bob: Arc::new(RwLock::new(Default::default())),
         last_smeared_timestamp: Arc::new(RwLock::new(0)),
         cmdline_sel_chat_id: Arc::new(RwLock::new(0)),
-        sentbox_thread: Arc::new(RwLock::new(unsafe {
-            dc_jobthread_init(
-                b"SENTBOX\x00" as *const u8 as *const libc::c_char,
-                b"configured_sentbox_folder\x00" as *const u8 as *const libc::c_char,
-                Imap::new(
-                    cb_get_config,
-                    cb_set_config,
-                    cb_precheck_imf,
-                    cb_receive_imf,
-                ),
-            )
-        })),
+        sentbox_thread: Arc::new(RwLock::new(dc_jobthread_init(
+            "SENTBOX\x00",
```

Comment:
this should remove the termination character, if you switch to rust strings

Reply:
Done.